components/ws-manager-api/go/core.pb.go

```diff
@@ -1,7 +1,3 @@
-// Copyright (c) 2020 TypeFox GmbH. All rights reserved.
-// Licensed under the GNU Affero General Public License (AGPL).
-// See License-AGPL.txt in the project root for license information.
-
 // Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
missing license header

Reply:
all over the place :/ we need some CI check for that (which I thought we had)